tests/python/contrib/test_arm_compute_lib/test_dense.py

```diff
@@ -101,7 +101,7 @@ def _get_qnn_model(
     out = relay.qnn.op.requantize(
         out,
         relay.const(input_sc * kernel_sc, "float32"),  # input scale
-        relay.const(input_zp * kernel_zp, "int32"),  # input zero point
+        relay.const(0, "int32"),  # input zero point
```

Comment:
What's the relevance of this change?

Reply:
Should be zero, as dense already counted all the input zero points in.